src/openrct2/core/FileIndex.hpp

```diff
@@ -204,7 +204,7 @@ template<typename TItem> class FileIndex
             JobPool jobPool;
             std::mutex printLock; // For verbose prints.
 
-            std::list<std::vector<TItem>> containers;
+            std::vector<std::vector<TItem>> containers;
```

Comment:
What is the reason behind this change?

Reply:
Internally, a vector is a pointer to a buffer. Having these pointers close to each other is a micro-optimization for when iterating over them.